app/boards/makerbase/mks_canable_v20/mks_canable_v20.conf

```diff
@@ -0,0 +1,20 @@
+# The three lines below are needed to reduce ROM usage
+CONFIG_LTO=y
+CONFIG_SIZE_OPTIMIZATIONS=y
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+
+CONFIG_USB_DEVICE_GS_USB_TIMESTAMP=y
+CONFIG_USB_DEVICE_GS_USB_TERMINATION=n
```

Comment:
These are automatically set based on the devicetree. Please remove these two lines.